context/app/static/js/components/Detail/Description/Description.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import SectionContainer from '../SectionContainer';
+import { StyledPaper, StyledInfoIcon } from './style';
+
+function Description(props) {
+  const { children } = props;
+  return (
+    <SectionContainer id="description">
```

Comment:
Sorry if you explained this before, but how is the `id` used? Are we pretty sure this will only occur only once on the page?

Reply:
Whoops, this code was taken from the `Preview` component. I'll remove the id.